base/applications/regedit/framewnd.c

```diff
@@ -283,6 +291,10 @@ static BOOL InitOpenFileName(HWND hWnd, OPENFILENAME* pofn)
     pofn->nMaxFileTitle = _MAX_PATH;
     pofn->Flags = OFN_EXPLORER | OFN_HIDEREADONLY;
     pofn->lpstrDefExt = L"reg";
+    if (bSave)
+        pofn->Flags |= OFN_OVERWRITEPROMPT | OFN_PATHMUSTEXIST;
+    else
+        pofn->Flags |= OFN_FILEMUSTEXIST | OFN_ENABLESIZING;
```

Comment:
Is it correct that OFN_ENABLESIZING is set only for reading, but not for saving? Seems illogical to me at first glance.

Reply:
![image](https://github.com/reactos/reactos/assets/2107452/c438c199-0db1-4fbc-8b14-eee4f0a75a74)
This Open/Save dialog is extended on Save. The extended area is not resizable.